WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -427,16 +427,8 @@ private extension QuickStartTourGuide {
         recentlyTouredBlog = nil
     }
 
-    func findNoticePresenter() -> NoticePresenter? {
```

Comment:
This really exemplifies how much cleaner this makes the code, eh?

Reply:
It does. The less code, the better. 😁